### ISSUE TITLE
QuickCheck Property types don't work in HTF (e.g. ==>)

### ISSUE THREAD

**Issue by jmtd** [skogsbaer/HTF#68]
Some QuickCheck stuff doesn't work in HTF.  For example using the `==>` operator:

```
prop_expandWindow2 s n = n < 10 ==> expandWindowPre1 n w == expandWindowPost1 w
…
    • Couldn't match expected type ‘Bool’ with actual type ‘Property’
```

This is likely true for `forAll` also, since those are the two combinators that are exceptions to the "properties must be Bool" rule in QuickCheck:

>  The result type of a property should be Bool, unless the property is defined using other combinators below. 
http://www.cse.chalmers.se/~rjmh/QuickCheck/manual.html

I'm not super-familiar with the internals ot HTF or QuickCheck but might this be resolved by adding an instance of the type class Testable for the class Property?

**Comment by jmtd** [comment]
This might be fixed now, at least I'm using it with 0.13.2.5.

**Comment by skogsbaer** [comment]
Added a test, bug was already fixed.
